### PR TITLE
Docker cache

### PR DESCRIPTION
**Pull request by 3nids**

### CODE REVIEW

File: docker-compose.yml.orig [hunk 1]
```diff
@@ -0,0 +1,59 @@
+version: '3.7'
```
Comment (olivierdalang) [inline docker-compose.yml.orig]: not sure what this file is ? (committed by mistake ?)

File: .github/workflows/test-migrations.yml [hunk 2]
```diff
@@ -2,21 +2,46 @@ name: Test Django Migrations
...
+      uses: docker/login-action@v1
+      with:
+        username: ${{ secrets.DOCKERHUB_USERNAME }}
+        password: ${{ secrets.DOCKERHUB_TOKEN }}
+
+    - name: Build Django image
+      uses: docker/build-push-action@v2
+      with:
+        context: .
+        file: docker/django/Dockerfile
```
Comment (olivierdalang) [inline .github/workflows/test-migrations.yml:35]: there's also the caddy Dockerfile that we must build, and maybe more in the future.

Isn't there a way to use the docker-compose setup with this approach ?

Comment (3nids) [reply .github/workflows/test-migrations.yml:35]: I don't think there is an action that build the compose + push all images.
Moreover, we might not want to push all images, only this one make sense to push for now.

Comment (olivierdalang) [reply .github/workflows/test-migrations.yml:35]: ok

**Review by olivierdalang — APPROVED** [review]
Ok for me (sorry for the conflict)

**Pull request merged**